### ISSUE TITLE
Crew complains of libraries missing, cannot reinstall crew

### ISSUE THREAD

**Issue by xinato** [skycocker/chromebrew#6102]
## Before you submit an issue
Please look up our issues, maybe your problem was mentioned before.

## Description
Provide a short description of your problem here.
I have Lenovo C630 chromebook.I installed crew in the past. I have not used it in a while. When I try to execute any crew command, I get

chronos@localhost / $ crew --vesion
Traceback (most recent call last):
        5: from /usr/local/bin/crew:10:in `<main>'
        4: from /usr/local/lib64/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        3: from /usr/local/lib64/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        2: from /usr/local/lib64/ruby/2.5.0/digest/sha2.rb:14:in `<top (required)>'
        1: from /usr/local/lib64/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/usr/local/lib64/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require': libssl.so.1.0.0: cannot open shared object file: No such file or directory - /usr/local/lib64/ruby/2.5.0/x86_64-linux/digest/sha2.so (LoadError)

When I try to re-istall crew I get:

chronos@localhost / $ curl -Ls git.io/vddgY | bash
bash: symbol lookup error: /usr/local/lib64/libreadline.so.8: undefined symbol: UP

How do I get around these issues? Can I uninstall crew and then reinstall it?

## Versions and system information
chromebrew: `crew --version`
This command fails, see above

chronos@localhost / $ cat /etc/lsb-release
CHROMEOS_ARC_ANDROID_SDK_VERSION=28
CHROMEOS_ARC_VERSION=7611540
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={495DCB07-E19A-4D7D-99B9-4710011A65B1}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={495DCB07-E19A-4D7D-99B9-4710011A65B1}
CHROMEOS_RELEASE_BOARD=nami-signed-mp-v5keys
CHROMEOS_RELEASE_BRANCH_NUMBER=82
CHROMEOS_RELEASE_BUILDER_PATH=nami-release/R92-13982.82.0
CHROMEOS_RELEASE_BUILD_NUMBER=13982
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=92
CHROMEOS_RELEASE_DESCRIPTION=13982.82.0 (Official Build) stable-channel nami 
CHROMEOS_RELEASE_KEYSET=mp-v5
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_UNIBUILD=1
CHROMEOS_RELEASE_VERSION=13982.82.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=13982.82.0

ChromeOS: `cat /etc/lsb-release`

Model: YOUR MODEL HERE
Lenovo Yoga Chromebook C630

Architecture: `uname -m`
Chronos@localhost / $ uname -m
x86_64

What I think needs to be done:
- [ ] step one
- [ ] step two

---

## That's it
Thank you for submitting your issue.
When done, please delete the parts of this template which you don't need or these, which are only for guidance.

**Comment by satmandu** [comment]
Easiest solution would be to just run the crew installer again.

`curl -Ls git.io/vddgY | bash`

**Comment by uberhacker** [comment]
I'll add a caveat.  Execute `rm -rf /usr/local` before attempting another reinstall also.  Of course, do this only if you don't have anything you need to save first.

**Comment by uberhacker** [comment]
On a freshly powerwashed i686 system, install failed at:
```
docbook_xsl: The DocBook XSL Stylesheets package contains XSL stylesheets. These are useful for performing transformations on XML DocBook files.
Precompiled binary available, downloading...
######################################################################## 100.0%
Docbook_xsl archive downloaded.
Unpacking archive using 'tar', this may take a while...
Performing pre-install...
Creating /usr/local/etc/xml/catalog
xmlcatalog: /usr/lib/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /usr/local/lib/libicui18n.so.69)
docbook_xsl failed to install: `xmlcatalog --noout --create /usr/local/etc/xml/catalog` exited with 1
```

**Comment by uberhacker** [comment]
I was able to install docbook_xsl with `yes | crew install gcc11 && yes | crew install docbook_xsl`.

**Comment by satmandu** [comment]
@xinato Did you get this resolved?

**Comment by xinato** [comment]
Thank you for your interest in my problem.

At first I was reluctant to delete all of /usr/local. I dragged my feet on
this, and finally did it.

After I deleted /usr/local/*, Chromebrew still failed to install because of
two circular links, /home/chronos/user/.config and
/home/chronos/user/.local. I deleted both of these links, and installation
completed. After installing, the .local link was recreated with a valid
reference, but the .config link was not recreated.

Again, many thanks for your kind support.

Best regards,

Chris Murphy
248.521.8704
